eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -70,6 +70,9 @@ stages:
         MatrixReplace:
```

Comment:
Do you want to add a MatrixFilter for data as well:

```
MatrixFilters:
  - TestFromSource=^$
```

Reply:
Do you mean this from a few lines above?

```
# Skip TestFromSource jobs for SDKType data
- ${{ if eq(parameters.SDKType, 'data') }}:
  - TestFromSource=^(?!true).*
```